docs/publish-module/binary-management/index.md

```diff
@@ -101,17 +103,16 @@ In Organization A's build profile workflow, after the [Export Build Artifacts](/
 #Bash script
 sudo npm install -g @appcircle/cli
 appcircle login --pat $ORG_B_PERSONAL_API_TOKEN
+#if ipa or aab is required change it to *.ipa or *.aab
+#--platform "ios" "android"
+#--markAsRc true/false Mark binary as release candidate automatically. [OPTIONAL] (default: false)
+#--summary "Release Notes" -> Release Notes (To add a release note to the app version, you need to mark the version as a release candidate.) [OPTIONAL]
 appcircle publish profile version upload \
   --platform <string> \
   --publishProfileId "$ORG_B_PUBLISH_PROFILE_ID" \
   --app "$AC_OUTPUT_DIR"/*.apk \
   --markAsRc <boolean> \
   --summary <string>
```

Comment:
Can you add this information as an infobox after the code section, and list it as bullet points?